### ISSUE TITLE
No module named 'aws_xray_sdk'

### ISSUE THREAD

**Issue by manifiserv** [awslabs/aws-lambda-powertools-python#1872]
### Expected Behaviour

I added the aws-lambda-powertools lib in my requirements.txt.

I used this same file from last one year with sam deployment.

Now I am getting error like 
[ERROR] Runtime.ImportModuleError: Unable to import module 'xxxx.xxxx': No module named 'aws_xray_sdk'Traceback (most recent call last):

In previously I didn't get the same error. Now only I am getting the error.

aws_xray_sdk is under the aws-lambda-powertools right?
No need to mention this xray lib to my requirements.txt right?

@aws
Please help me to resolve this issue?

Thanks,

### Current Behaviour

Now I am getting error like 
[ERROR] Runtime.ImportModuleError: Unable to import module 'xxxx.xxxx': No module named 'aws_xray_sdk'Traceback (most recent call last):

In previously I didn't get the same error. Now only I am getting the error.

### Code snippet

```python
When I import this line, I am getting error

from aws_lambda_powertools import Logger, Metrics, Tracer
```

### Steps to Reproduce

I am raising the issue

### AWS Lambda Powertools for Python version

latest

### AWS Lambda function runtime

3.9

### Packaging format used

Lambda Layers

**Comment by leandrodamascena** [comment]
> ### Expected Behaviour
> I added the aws-lambda-powertools lib in my requirements.txt.
> 
> I used this same file from last one year with sam deployment.
> 
> Now I am getting error like [ERROR] Runtime.ImportModuleError: Unable to import module 'xxxx.xxxx': No module named 'aws_xray_sdk'Traceback (most recent call last):
> 
> In previously I didn't get the same error. Now only I am getting the error.
> 
> aws_xray_sdk is under the aws-lambda-powertools right? No need to mention this xray lib to my requirements.txt right?
> 

Hi @manifiserv! I see you are using the same code as 1 year ago and it is not working. The problem with this is that 1 year ago our most recent version pointed to a 1.x version, but we worked a lot since then and released the 2.0 version. In version 2.0 the xray SDK is not included by default, as this would introduce a size overhead even for those who would not be using Tracer.
To solve this problem, just put in your requirements.txt the tracer dependency using `aws-lambda-powertools[tracer]` or all dependencies using `aws-lambda-powertools[all]`.
However, we suggest you take a look at our [upgrade guide](https://awslabs.github.io/aws-lambda-powertools-python/2.7.0/upgrade/) to see if you have any compatibility with the current code.

More information about v2 announcement here: https://github.com/awslabs/aws-lambda-powertools-python/releases/tag/v2.0.0

Thank you

**Comment by manifiserv** [comment]
@leandrodamascena 
Thank you..
